CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Current Master
 
+- Turn off auto faucet
```

Comment:
This note could be more user-friendly. This is meaningful to us, but does not describe the change to a user. How about something like:
`- No longer automatically request 1 ropsten ether for the first account in a new vault.`?